File: EJ6/Circulo.php
<?php
    class Circulo {
        public $radio;

        public function calcularArea() {
            $total = $this->radio * $this->radio * 3.14;
            echo "El área del circulo es: " .$total;
        }
    }

    $circulo = new Circulo();
    $circulo->radio = "5";

    $circulo->calcularArea();
?>

File: Ejercicio4/EJ10/misPaquetes/Circulo.java
package misPaquetes;

public class Circulo extends Figura{
	public static final double PI = Math.PI;
	double radio;
	
	public Circulo(double radio) {
		this.radio = radio;
	}
	
	@Override
	public void calcularArea() {
		double resultado = PI * radio * radio;
		System.out.println("El resultado es: " + resultado);
	}
}
